src/lib/db/patterns.ts

```diff
@@ -0,0 +1,9 @@
+import { NavResource } from '../v2/interfaces/resources'
+
+export function getResourceId(source: NavResource) {
+  if (source.type === 'doc') {
```

Comment:
I don't think we need this function or this conditional statement since it i returning same value